### DISCUSSION TITLE
Use of 320 x 480 display

### DISCUSSION THREAD

**Discussion by hugowaals** [Bodmer/TFT_eSPI#1734] [Compatible displays and setup files]
Good morning,

I assume it's a simple question for you but can I also use an ILI9341 320 x 480 display?

thank you in advance,
Hugo

**Comment by Bodmer** [comment]
I am not aware of ILI9341 screens being more than 240x320 pixels. That is a pixel limit on that display controller. Post a vendors link as it may be wrong.

**Comment by hugowaals** [comment]
Sorry sorry sorry, my fault, it’s the ILI9488 controler (on AliExpress)
Thanks for the reply 

Best regards 

Hugo

**Reply by Bodmer** [reply]
Yes, that controller is supported with a SPI interface.
